scrapi/harvesters/biomedcentral.py

```diff
@@ -6,134 +6,128 @@
-            'additionalName': name.middle,
-            'familyName': name.last,
-            'sameAs': [],
-        }
-        contributor_list.append(contributor)
+    for url in all_uris:
+        if 'dx.doi.org' in url:
+            processed_uris['objectUris'].append(url)
 
-    return contributor_list
+    return processed_uris
 
 
 class BiomedCentralHarvester(JSONHarvester):
     short_name = 'biomedcentral'
     long_name = 'BioMed Central'
-    url = 'http://www.biomedcentral.com/'
+    url = 'http://www.springer.com/us/'
     count = 0
 
-    URL = 'http://www.biomedcentral.com/search/results?terms=*&format=json&drpAddedInLast={}&itemsPerPage=250'
+    base_url = 'http://api.springer.com/meta/v1/json'
+    URL = furl(base_url)
+    URL.args['api_key'] = settings.SPRINGER_API_KEY
+    URL.args['p'] = 100
 
     @property
     def schema(self):
         return {
-            'contributors': ('/authorNames', process_contributors),
-            'uris': {
-                'canonicalUri': '/articleFullUrl',
-                'objectUris': ('/doi', 'http://dx.doi.org/{}'.format),
-                'providerUris': ('/articleFullUrl', '/abstractPath', lambda x, y: [x, y])
-            },
-            'title': ('/bibliographyTitle', '/blurbTitle', lambda x, y: x or y),
-            'providerUpdatedDateTime': ('/published Date', datetime_formatter),
-            'description': '/blurbText',
+            'contributors': (
+                '/creators',
+                compose(
+                    default_name_parser,
+                    lambda authors: [author['creator'] for author in authors]
+                )
+            ),
+            'uris': ('/url', process_urls),
+            'title': '/title',
+            'providerUpdatedDateTime': ('/publicationDate', datetime_formatter),
+            'description': '/abstract',
             'freeToRead': {
-                'startDate': ('/is_free', '/published Date', lambda x, y: y if x else None)
+                'startDate': ('/openaccess', '/publicationDate', lambda x, y: y if x == 'true' else None)
             },
+            'publisher': {
+                'name': '/publisher'
+            },
+            'subjects': ('/genre', lambda x: [x] if x else []),
             'otherProperties': build_properties(
-                ('imageURL', '/imageUrl', {'description': 'a image url'}),
-                ('type', '/type'),
-                ('isOpenAccess', '/isOpenAccess'),
-                ('articleUrl', '/articleUrl'),
-                ('articleFullUrl', '/articleFullUrl'),
-                ('isFree', '/isFree'),
-                ('isHighlyAccessed', '/isHighlyAccessed'),
-                ('status', '/status'),
-                ('abstractPath', '/abstractPath'),
-                ('journal Id', '/journal Id'),
-                ('article_host', '/article_host'),
-                ('longCitation', '/longCitation'),
-                ('is_subscription', '/is_subscription')
+                ('url', '/url'),
```

Comment:
is this URL captured in one of our URL fields? If so we can remove it from otherProperties